x-pack/metricbeat/docs/modules/aws.asciidoc

```diff
@@ -0,0 +1,65 @@
+////
```

Comment:
As I can't comment on the image above, comment is here. The image above probably should go outside the x-pack directory into docs as all docs are in one place. As discussed before, we need to update docs_collector script to also collect the docs from the x-pack modules into the non-xpack docs directory: https://github.com/elastic/beats/blob/master/metricbeat/scripts/docs_collector.py

Reply:
I tried to move `x-pack/metricbeat/docs/modules/aws.asciidoc` into non-xpack directory `beats/metricbeat/docs/modules/aws.asciidoc` but seems like CI doesn't like it hmm https://travis-ci.org/elastic/beats/jobs/467148471